changelog/fix16997.dd

```diff
@@ -0,0 +1,17 @@
+-preview=intpromote switch has now been set to true by default now.
```

Comment:
-preview=intpromote switch is now set by default.

Reply:
Done